op-challenger/game/fault/contracts/faultdisputegame.go

```diff
@@ -231,6 +231,15 @@ func (f *FaultDisputeGameContract) addGlobalDataTx(ctx context.Context, data *ty
 	return oracle.AddGlobalDataTx(data)
 }
 
+func (f *FaultDisputeGameContract) GetWithdrawals(ctx context.Context, block rpcblock.Block, gameAddr common.Address, recipients ...common.Address) ([]*WithdrawalRequest, error) {
+	defer f.metrics.StartContractRequest("GetWithdrawals")()
+	delayedWETH, err := f.GetDelayedWETH(ctx)
```

Comment:
nit: Can we make `GetDelayedWETH` private now? Not an issue if it is being used by something else.